transformer_engine/pytorch/cpp_extensions/fused_attn.py

```diff
@@ -1106,6 +1151,13 @@ def fused_attn_bwd(
                 type of the bias; {"no_bias", "pre_scale_bias", "post_scale_bias", "alibi"}
     attn_mask_type: str, default = "padding"
                 type of the attention mask; {"padding", "causal", "padding_causal", "no_mask"}
+    window_size: Tuple[int, int], default = (-1, -1)
+                sliding window size for local attention, where query at position i attends to keys
+                in [i + seqlen_k - seqlen_q - window_size[0], i + seqlen_k - seqlen_q
+                + window_size[1]] inclusive. Special cases (-1, -1) and (-1, 0) mean no sliding
```

Comment:
Why there are two special cases for the same "no sliding". This seems hard to maintain.

Reply:
The two special cases are no sliding window and causal mask, not just no sliding window. I added more words to describe them: 

Special cases (-1, -1) and (-1, 0) mean no sliding window and causal mask specifically. Both `causal` and 
`causal_bottom_right` masks map to `window_size = (-1, 0)` and Transformer Engine distinguishes them based on `attn_mask_type`. Similar to `attn_mask_type`, `window_size` can be overridden by `window_size` in `forward` as well.